IMS/FrontEnd/Program.cs

```diff
@@ -7,7 +7,12 @@
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
-builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri("https://localhost:7216") });
+string? serverBaseURI = builder.Configuration["ApiHostName"];
```

Comment:
We typically just use var here for local variable assignment:

```suggestion
var serverBaseURI = builder.Configuration["ApiHostName"];
```